lighthouse-core/audits/errors-in-console.js

```diff
@@ -78,7 +78,7 @@ class ErrorLogs extends Audit {
     /** @type {LH.Audit.Details.Table['headings']} */
     const headings = [
       {key: 'url', itemType: 'url', text: str_(i18n.UIStrings.columnURL)},
-      {key: 'description', itemType: 'code', text: str_(UIStrings.columnDesc)},
+      {key: 'description', itemType: 'text', text: str_(UIStrings.columnDesc)},
```

Comment:
I think I'd prefer to keep this one as code since in my experience these are less wall of text than deprecation explanations, but I don't feel terribly strongly.

Reply:
yeah, I probably agree with this (and it matches how it's displayed in the devtools console), but without auto code markdown for the symbols in each of these, display won't be ideal, mono or not